### ISSUE TITLE
BartForSequenceClassification이 올바르게 동작하지 않습니다.

### ISSUE THREAD

**Issue by dha8102** [SKT-AI/KoBART#21]
## 🐛 Bug

BartForSequenceClassification 이 제대로 동작하지 않습니다. Tutorial에서 model만 살짝 바꿨는데 에러가 나네요.
Original(English) Bart는 잘 동작하는데, KoBART는 안 되네요.. eos token 관련 문제인 것 같은데, 확인 부탁드립니다.
## To Reproduce

from transformers import BartTokenizer, BartForSequenceClassification
from kobart import get_pytorch_kobart_model, get_kobart_tokenizer
from transformers.models.bart.modeling_bart import *

tokenizer = BartTokenizer.from_pretrained("facebook/bart-base")
model = BartForSequenceClassification.from_pretrained("facebook/bart-base")

inputs = tokenizer(["Hello, my dog is cute"], return_tensors="pt")
labels = torch.tensor([1]).unsqueeze(0)  # Batch size 1
outputs = model(inputs['input_ids'], labels=labels)
print(outputs.logits)

kobart_tokenizer = get_kobart_tokenizer()
ko_model = BartForSequenceClassification.from_pretrained(get_pytorch_kobart_model())

inputs = kobart_tokenizer(['안녕하세요.'], return_tensors='pt')
labels = torch.tensor([1]).unsqueeze(0)  # Batch size 1
ko_model(inputs['input_ids'], labels=labels)

버그를 재현하기 위한 재현절차를 작성해주세요.

1. BART와 KoBART 설치 후 실행

## Expected behavior

정상적으로 outputs에 해당하는 값이 나와야 하는데...안 나오네요

## Environment

Colab과 Local 모두에서 테스트
Local 환경 
Package                 Version
----------------------- ---------
absl-py                 1.0.0
cachetools              5.0.0
certifi                 2021.10.8
charset-normalizer      2.0.11
click                   8.0.3
colorama                0.4.4
cycler                  0.11.0
filelock                3.4.2
fonttools               4.29.1
google-auth             2.6.0
google-auth-oauthlib    0.4.6
grpcio                  1.43.0
huggingface-hub         0.4.0
idna                    3.3
importlib-metadata      4.10.1
joblib                  1.1.0
kiwisolver              1.3.2
Markdown                3.3.6
matplotlib              3.5.1
mkl-fft                 1.3.1
mkl-random              1.2.2
mkl-service             2.4.0
nltk                    3.6.7
numpy                   1.21.2
oauthlib                3.2.0
olefile                 0.46
packaging               21.3
Pillow                  8.4.0
pip                     21.2.2
protobuf                3.19.4
pyasn1                  0.4.8
pyasn1-modules          0.2.8
pyparsing               3.0.7
python-dateutil         2.8.2
PyYAML                  6.0
regex                   2022.1.18
requests                2.27.1
requests-oauthlib       1.3.1
rouge-score             0.0.4
rsa                     4.8
sacremoses              0.0.47
setuptools              58.0.4
six                     1.16.0
tensorboard             2.8.0
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.1
tokenizers              0.11.4
torch                   1.10.2
torchaudio              0.10.2
torchvision             0.11.3
tqdm                    4.62.3
transformers            4.16.2
typing-extensions       3.10.0.2
urllib3                 1.26.8
Werkzeug                2.0.2
wheel                   0.37.1
wincertstore            0.2
zipp                    3.7.0

**Comment by bage79** [comment]
@dha8102 님, 안녕하세요.

`kobart_tokenizer = get_kobart_tokenizer()` 에서 만든 tokenizer는 `<s>` 및  `</s>` 토큰을 자동으로 입력해주지 않아서 생긴 문제로 보입니다.

```python
inputs = kobart_tokenizer(['<s>안녕하세요.</s>'], return_tensors='pt')
```
위처럼 입력에 BOS, EOS 토큰을 붙여 주시면, 아래처럼 정상 출력하는 것을 확인하였습니다.
확인 부탁드립니다.
![image](https://user-images.githubusercontent.com/304911/154927387-e0b256a6-c884-4230-b415-c6fa28bdba91.png)

**Comment by bage79** [comment]
문제가 해결된 것으로 판단되어 이 이슈는 Close 합니다.
